examples_test.go

```diff
@@ -0,0 +1,38 @@
+package flyway_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/CyberOwlTeam/flyway"
+	"log"
+)
+
+func ExampleRunContainer() {
+	// runFlywayContainer {
+	ctx := context.Background()
+
+	flywayContainer, err := flyway.RunContainer(ctx)
```

Comment:
We recommend passing the image always here, so users know what image they are using. We are discussing if pushing the image to the signature of the RunContainer function 🤔

Reply:
I did the same in the test also